ConwaysGameOfLife/Universe.cs

```diff
@@ -53,40 +53,33 @@ public bool AreAllCellsDead()
 
         public List<Location> GetCellNeighbourLocations(Cell cell)
         {
-            var neighbourLocations = new List<Location>();
-            neighbourLocations.Add(new Location(cell.Location.X - 1, cell.Location.Y - 1));//0,0
-            neighbourLocations.Add(new Location(cell.Location.X - 1, cell.Location.Y));// 0,1
-            neighbourLocations.Add(new Location(cell.Location.X - 1, cell.Location.Y + 1));//0,2
-            neighbourLocations.Add(new Location(cell.Location.X, cell.Location.Y - 1));//1,0
-            neighbourLocations.Add(new Location(cell.Location.X, cell.Location.Y + 1));//1,2
-            neighbourLocations.Add(new Location(cell.Location.X + 1, cell.Location.Y - 1));//2,0
-            neighbourLocations.Add(new Location(cell.Location.X + 1, cell.Location.Y));//2,1
-            neighbourLocations.Add(new Location(cell.Location.X + 1, cell.Location.Y + 1));//2,2
+            var neighbourLocations = new List<Location>() {
+                new Location(cell.Location.X - 1, cell.Location.Y - 1),     //0,0
+                new Location(cell.Location.X - 1, cell.Location.Y),         //0,1
+                new Location(cell.Location.X - 1, cell.Location.Y + 1),     //0,2
+                new Location(cell.Location.X, cell.Location.Y - 1),         //1,0
+                new Location(cell.Location.X, cell.Location.Y + 1),         //1,2
+                new Location(cell.Location.X + 1, cell.Location.Y - 1),     //2,0
+                new Location(cell.Location.X + 1, cell.Location.Y),         //2,1
+                new Location(cell.Location.X + 1, cell.Location.Y + 1)      //2,2
+            };
+            
             return neighbourLocations;
         }

```

Comment:
note: we still need to account for neighbours of cells that touch the universe boundary/boundaries